### ISSUE TITLE
req.options.action is wrong

### ISSUE THREAD

**Issue by pine613** [balderdashy/sails#2468]
Hello. I'm using Sails.js (v0.10.5). I found a bug that `req.options.action` is different depending on the route.

In `api/controller/TestController.js`:
```js
module.exports = {
  testAction: function (req, res) {
    res.send(req.method + ': ' + req.options.action);
  }
};
```

In `config/routes.js`:
```js
module.exports.routes = {
  'get /test/testAction': 'TestController.testAction'
};
```

In this case, execute `$ sails lift` and following commands:

```
$ curl -X GET localhost:1337/test/testAction
GET: testaction
$ curl -X POST localhost:1337/test/testAction
POST: testAction
```

I do not know that it is normal operation but I think that it may annoy for developers. I think that it is so good if `req.options.action` are unified. Thank you.

**Comment by martingg88** [comment]
i have experienced this issue as well. hope it can be solved as soon as possible

**Comment by CWyrtzen** [comment]
Is this a bug?

**Comment by sailsbot** [comment]
Thanks for posting, @pine613.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
